src/deflate/zopfli_oxipng.rs

```diff
@@ -6,10 +6,10 @@ pub fn deflate(data: &[u8], iterations: NonZeroU8) -> PngResult<Vec<u8>> {
 
     let mut output = Vec::with_capacity(max(1024, data.len() / 20));
```

Comment:
While we're here, I noticed this vec with initially small capacity can negatively impact performance in some cases as it needs to reallocate during compression. I think we could increase to maybe 1/4, or even just 1 as is used (by necessity) for libdeflate.

Reply:
That sounds good, I'll change it!